namespaces/live.cloud-platform.service.justice.gov.uk/interventions-vision/00-namespace.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "interventions-vision"
```

Comment:
Can you prepend this with hmpps please, so it matches with other HMPPS namespaces.